pkg/services/govmomi/metadata/metadata_suite_test.go

```diff
@@ -141,7 +141,7 @@ func configureSimulatorAndContext() (err error) {
 		return
 	}
 
-	vmCtx = fake.NewVMContext(fake.NewControllerContext(fake.NewControllerManagerContext()))
+	vmCtx = fake.NewVMContext(ctx, fake.NewControllerContext(fake.NewControllerManagerContext()))
```

Comment:
Because this is a helper function: we should pass in `ctx` as a parameter in the function instead of using the package variable.


```go
func configureSimulatorAndContext(ctx context.Context) (err error) {
```